Security/Firewall/Message/LtiPlatformMessageAuthenticationListener.php

```diff
@@ -71,6 +71,15 @@ public function authenticate(RequestEvent $event): void
         $token->setAttribute('firewall_config', $this->firewallMap->getFirewallConfig($request));
 
         $this->storage->setToken($this->manager->authenticate($token));
+    }
+
+    private function getJWTFrom(Request $request): ?string
+    {
+        $JWTFromQuery = $request->query->get('JWT');
+        if (null !== $JWTFromQuery) {
+            return $JWTFromQuery;
```

Comment:
Method names MUST be `camelCased()` according to PSR-1.
The variables should also be `$camelCased` as the rest of them in the scope are.

The acronyms are never capitalized when camelCased.
```suggestion
    private function getJwtFrom(Request $request): ?string
    {
        $jwtFromQuery = $request->query->get('JWT');
        if (null !== $jwtFromQuery) {
            return $jwtFromQuery;
```

Reply:
This is a good discussion, because JWT is not a name, but I will change it.

Reply:
It is an acronym :)